<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1628</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kicakLibCreator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Octopart</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="6">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4" rowspan="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>search</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2">
         <widget class="QComboBox" name="comboBox">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="7">
         <widget class="QTableWidget" name="tableOctopartResult"/>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="cbtn_exact_match">
          <property name="text">
           <string>exact match only</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>KiCad input library</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="7" column="0">
         <widget class="QListWidget" name="list_input_libraries"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Library</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Devices</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QListWidget" name="list_input_devices"/>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QScrollArea" name="scrollQuickLink">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>120</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1055</width>
             <height>118</height>
            </rect>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QWidget" name="widget" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="lbl_sourceOctopart">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Selected Octopart MPN:</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>KiCad lib creation</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="14" column="2" colspan="4">
         <widget class="QComboBox" name="cmb_targetLibrary">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="btn_editRule">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QLineEdit" name="edt_targetDescription"/>
        </item>
        <item row="6" column="4">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Download</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>F5 (mpn)</string>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>F4 (ID)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>F0 (Designator)</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLineEdit" name="edt_targetMPN"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>F2 (Footprint)</string>
          </property>
         </widget>
        </item>
        <item row="13" column="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Library:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLineEdit" name="edt_targetID">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>F6 (manufacturer)</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLineEdit" name="edt_targetDatasheet">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>F7 (Display value)</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QLineEdit" name="edt_targetDisplayValue"/>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="edt_targetName"/>
        </item>
        <item row="4" column="2">
         <widget class="QComboBox" name="cmb_targetFootprint">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Def/F1 (name/value)</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLineEdit" name="edt_targetManufacturer"/>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="btn_applyRule">
          <property name="text">
           <string>apply</string>
          </property>
         </widget>
        </item>
        <item row="15" column="4" colspan="2">
         <widget class="QPushButton" name="btnCreatePart">
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>F3 (datasheet)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Source symbol:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3" colspan="2">
         <widget class="QLabel" name="lbl_targetFootprint">
          <property name="text">
           <string>0402</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="edt_targetDesignator"/>
        </item>
        <item row="6" column="5">
         <widget class="QPushButton" name="btn_show_datasheet">
          <property name="text">
           <string>Show</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Rule:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QComboBox" name="cmb_targetRuleName">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lblSourceDevice">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1628</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Ex&amp;tras</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuRules">
    <property name="title">
     <string>&amp;Rules</string>
    </property>
    <addaction name="actionEdit"/>
   </widget>
   <addaction name="menuExtras"/>
   <addaction name="menuRules"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOptions">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>&amp;Edit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
